ega-charts/localega/templates/doa-secret.yaml

```diff
@@ -1,11 +1,11 @@
 {{- if .Values.doa.deploy }}
-{{- $c4gh_key := .Files.Glob "config/key.1.c4gh.sec" }}
+{{- $c4gh_key := .Files.Glob "config/ega_key.c4gh.sec" }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ template "localega.name" . }}-doa-secret
 data:
 {{ ( $c4gh_key ).AsSecrets | trim | indent 2 }}
-  key.pass: {{ .Values.secrets.c4gh_passphrase | quote | trimall "\"" | b64enc }}
+  key.pass: {{ .Values.secrets.ega_c4gh_passphrase | quote | trimall "\"" | b64enc }}
```

Comment:
This change is not related to NFS

Reply:
true, but the helm linter will fail because the deploy-init is updated and it cannot find that key, unless, I wait for another branch to be merged first.

Reply:
rebased to milestone_8